### ISSUE TITLE
Multiphysics MSRE - Griffin-Pronghorn stand alone. 

### ISSUE THREAD

**Issue by mtnwalker** [idaholab/virtual_test_bed#311]
## Model Description
This model is a multiphysics RZ model of the MSRE using only Griffin and Pronghorn. Can run on a work station.

## Project description
This model was developed by Mauricio Tano. Funded by the NEAMS program. Future work will incorporate the Domain Overlapping capability.

## Impact
This is a valuable contribution to modeling the MSRE with RZ resolution which will be used by the NEAMS program for further chemical analyses development in FY24.

**Comment by hapfang** [comment]
Please be aware that the mesh file mentioned in the documentation and input files is the Moose-generated mesh. However, in the mesh/ subdirectory, only a Moose mesh input file is present. If attempting to run the multiphysics model, an error may occur, stating "../mesh/mesh_in.e, error = No such file or directory."

**Comment by GiudGiud** [comment]
We dont want to check in the mesh file if we can re-generate it.
The documentation should hopefully mention to generate the mesh before running the model

**Comment by hapfang** [comment]
Agree, a note on how to generate the mesh from the input file should be sufficient.
